src/pb_stub.cc

```diff
@@ -1670,6 +1684,7 @@ PYBIND11_EMBEDDED_MODULE(c_python_backend_utils, module)
           py::return_value_policy::reference_internal)
       .def("request_id", &InferRequest::RequestId)
       .def("correlation_id", &InferRequest::CorrelationId)
+      .def("correlation_id_string", &InferRequest::CorrelationIdString)
```

Comment:
Let's only have a single `correlation_id` field and it should return string or int depending on which one has been set.